optional-integration-analysis/merge-sce.R

```diff
@@ -72,6 +77,15 @@ if(is.null(opt$input_metadata_tsv)){
   input_metadata <- readr::read_tsv(opt$input_metadata_tsv)
 }
 
+# Check that integration group is provided
+if(is.null(opt$integration_group)){
+  stop("The name of the corresponding integration group is missing.")
```

Comment:
```suggestion
  stop("The `integration_group` is missing and required for identifying which objects should be merged together.")
```
Just clarifying the error message a little bit.